backend/db/refresh.py

```diff
@@ -68,8 +68,9 @@ def refresh_db(
             # todo: when ready, will use all_new_objects_enclave_to_db() instead of csets_and_members_enclave_to_db()
             new_data: bool = csets_and_members_enclave_to_db(con, since, schema=schema)
         except Exception as err:
-            update_db_status_var('last_refresh_result', 'error', local)
             print(f"Database refresh incomplete; exception occurred. Tallying counts and exiting.", file=sys.stderr)
+            update_db_status_var('last_refresh_result', 'error', local)
+            reset_temp_refresh_tables(schema)
```

Comment:
## Reset tables upon failure
I recently added `reset_temp_refresh_tables()` to my vocab/counts PR.